tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
Does this run on browsers? You might get away with targetting higher versions since node supports lots of new features.

However, es5 is a good default if you're not sure.

Reply:
In short, I think ES5 is the right choice for now.

Helmet hasn't had a breaking change since Node 4, which has some [holes in its ES6 support](https://kangax.github.io/compat-table/es6). Node 4 is retired so no one _should_ be using it, but just in case, I want to avoid ES6. (One could argue that I shouldn't care about a retired Node version, but that's a separate issue.)

Helmet's next major version bump will break Node <8 (https://github.com/helmetjs/helmet/issues/146) and then we can finally use ES6 then.